grpc-proto/proto/query.proto

```diff
@@ -116,95 +117,90 @@ message Values {
 
 message Payload {
   Type type = 1;
-  enum Type { // Do we need an enum here or can we detect the type from Any?
-    TYPE_CQL = 0;
-    // TYPE_JSON = 1;
-    // TYPE_AVRO = 2;
+  enum Type {
+    CQL = 0;
+    // JSON = 1;
+    // AVRO = 2;
     // ...
   }
-  google.protobuf.Any value = 2;
+  google.protobuf.Any data = 2;
 }
 
 message QueryParameters {
   google.protobuf.StringValue keyspace = 1;
-  // TODO: Output payload type? Or is it the same as the input and CQL by default?
-  Payload payload = 2;
-  ConsistencyValue consistency = 3;
-  google.protobuf.Int32Value page_size = 4;
-  google.protobuf.BytesValue paging_state = 5;
-  bool tracing = 6;
-  bool skip_metadata = 7;
-  google.protobuf.Int64Value timestamp = 8;
-  ConsistencyValue serial_consistency = 9;
-  google.protobuf.Int32Value now_in_seconds = 10;
+  ConsistencyValue consistency = 2;
+  google.protobuf.Int32Value page_size = 3;
+  google.protobuf.BytesValue paging_state = 4;
+  bool tracing = 5;
+  bool skip_metadata = 6;
+  google.protobuf.Int64Value timestamp = 7;
```

Comment:
nit: Could you add a comment about the expected representation of timestamp values in this case?

Reply:
Will followup in https://github.com/stargate/stargate/issues/990.